api/server/models/optionModel.js

```diff
@@ -1,7 +1,10 @@
 var mongoose = require("mongoose");
 
 var optionSchema = mongoose.Schema({
-  time: Date,
+  time: {
```

Comment:
I think to keep consistency, you should rename to startTime and endTime (or if you think it should be 'start' and 'end' then Leo can change his attribute names in the Meeting model). Also, I don't think you need to keep them in the 'time' object, just make them direct children of Option.

Reply:
I like it this way better because a start and end time intuitively seem like something to be grouped together. I can change 'time' to 'timeline' or 'duration' though. The reason I don't like startTime or endTime is that even though that's how we'll use this data, this is a date and not exactly the time; we'll also be pulling month and day data from these dates so I don't think we should call these time

Reply:
That's a valid point. I guess my reasoning for keeping them separate is that there's only one other attribute for an Option, so it seems unnecessary to have to go through an attribute to get to startTime and endTime. Either way, if you want to keep it this way it should be formatted the same way on Leo's Meeting models so that should be discussed with him.

Reply:
Also, I'm fine with keeping it as 'start' and 'end' instead of 'startTime' and 'endTime'. I do want to say though that even though it's called a Date, it's really an specific point in time (specifically the time in milliseconds "since midnight on January 1, 1970, UTC") so calling it a Date (which isn't your fault) isn't really accurate.